operations/tempo-mixin/runbook.md

```diff
@@ -1,8 +1,7 @@
 # Runbook
 
-This document should help with remediating operational issues in Tempo.
+This document should help with remediation of operational issues in Tempo.
 
-## TempoRequestErrors
```

Comment:
i think this might be linked by internal docs in deployment tools, please double check

Reply:
No mentions outside of the mixin that I see.